widgets/range-slider.js

```diff
@@ -0,0 +1,67 @@
+var React = require('react');
+
+var utils = require('../lib/utils.js');
+
+/**
+ * Instantiate a slider based on a numeric attribute
+ * @param  {String|DOMElement} options.container Valid CSS Selector as a string or DOMElement
+ * @param  {String} options.facetName Name of the attribute for faceting
+ * @param  {Boolean|Object} [options.tooltips=true] Should we show tooltips or not.
+ * You can also provide tooltips: {format: function(formattedValue, rawValue) {return string}} option object
+ * @return {Object}
+ */
+function rangeSlider({
+    container = null,
+    facetName = null,
+    tooltips = true
+  }) {
+  var Slider = require('../components/Slider');
+
+  var containerNode = utils.getContainerNode(container);
+
+  return {
+    getConfiguration: () => ({
+      disjunctiveFacets: [facetName]
+    }),
+    _currentValues: {
+      min: -Infinity,
+      max: Infinity
+    },
+    _refine(helper, stats, newValues) {
+      var min = helper.state.getNumericRefinement(facetName, '>=');
+      var max = helper.state.getNumericRefinement(facetName, '<=');
+
+      this._currentValues = {
+        min: Math.max(min && min.length && min[0] || 0, stats.min),
```

Comment:
Why zero? Why not -Infinity?

Min and max takes an arbitrary number of values. So what about using : 
```js
min : min && min.length ? Math.max.apply(Math, min.concat(stats.min)) : stats.min;
```